packages/@vuepress/theme-default/styles/index.styl

```diff
@@ -11,10 +11,10 @@ html, body
   background-color #fff
 
 body
-  font-family -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen, Ubuntu, Cantarell, "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif
+  font-family -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen, Ubuntu, Cantarell, "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol"
```

Comment:
it is something just for emoji can you explain why implement this ?

Reply:
Yes, adding those to your font stack allows Emoji to be rendered using the devices native library anywhere in the app and is common in font stacks.